pyproject.toml

```
@@ -143,6 +144,7 @@ scrublet = ["scrublet"]  # Doublet detection
 # Acceleration
 rapids = ["cudf>=0.9", "cuml>=0.9", "cugraph>=0.9"]  # GPU accelerated calculation of neighbors
 dask = ["dask[array]!=2.17.0"]  # Use the Dask parallelization engine
+dask-ml = ["dask-ml"]  # Dask-ML for sklearn-like API
```

This makes sure that our version constraint gets obeyed when installing `scanpy[dask-ml]`

```suggestion
dask-ml = ["dask-ml", "scanpy[dask]"]  # Dask-ML for sklearn-like API
```